static/app/utils/profiling/formatters/stackMarkerToHumanReadable.ts

```diff
@@ -0,0 +1,19 @@
+export function stackMarkerToHumanReadable(marker: JSSelfProfiling.Marker): string {
+  switch (marker) {
+    case 'gc':
+      return 'Garbage Collection';
```

Comment:
If this gets displayed we should make it translatable:
```suggestion
      return t('Garbage Collection');
```
Same for all other strings in this PR.

t comes from `import {t} from 'sentry/locale';`